fourmolu.yaml

```diff
@@ -0,0 +1,10 @@
+indentation: 4
+comma-style: leading
+import-export-comma-style: leading
+indent-wheres: false
+record-brace-space: true
+diff-friendly-import-export: true
+respectful: true
+haddock-style: multi-line
+newlines-between-decls: 1
+fixities: []
```

Comment:
Please do match the styleguide here
https://github.com/mlabs-haskell/styleguide#code-formatting